src/thing_doors.h

```diff
@@ -44,13 +44,10 @@ struct Thing;
 struct DoorStats { // sizeof = 8
     unsigned short slbkind;
     long health;
-    unsigned short field_6;
+    unsigned short open_speed;
 };
```

Comment:
can't the DoorStats struct be removed entirely as it's replaced with DoorConfigStats everywhere

Reply:
Done.